packages/gatsby-plugin-page-creator/src/__tests__/gatsby-node.js

```diff
@@ -1,5 +1,6 @@
 const validatePath = require(`../validate-path`)
 const createPath = require(`../create-path`)
+const ignorePath = require(`../ignore-path`)
```

Comment:
Can we move those tests to separate test file (`ignore-path.js`)? Added tests check `ignorePath` specifically - so moving them to separate file should make it cleaner?

Reply:
Done!